### DISCUSSION TITLE
Vlan for Voice configuration for $200

### DISCUSSION THREAD

**Discussion by Thaad-QR** [Anime4000/RTL960x#343] [Help]
Hi
I replaced my Gpon Huawei router with ODI Realtek DFP-34X-2C2 and its work fine by creating Vlan10 for internet with PPPoE on sfp port.
am still suffering from configuration issue with Vlan30 for Voice or VoIP the scenario is:
![EymCq2](https://github.com/user-attachments/assets/32131994-940b-43ce-b7fd-e909a22ad6da)
this is address list, dhcp, etc for your vlan30
IP Acquisition Mode:DHCP
IP Address/Subnet Mask:xx.xxx.xxx.xxx/255.255.224.0
Gateway: xx.xxx.xxx.x
DNS Servers: xxx.xx.xxx.xx,xxx.xx.xxx.xx

if i have to hire someone to do it fro me online i have no problem for that I will pay him $200 via paypal
best regards

**Comment by ToTheCLI** [comment]
Check your telegram, I have successfully accomplished this with STC SIP VOIP in KSA

**Comment by Thaad-QR** [comment]
any help please

**Comment by billbarni** [comment]
I have similar need...

I have a working DFP-34X-2C2 (and 2C3) for internet. But VOIP never authenticates.
I have it behind a OpnSense that grabs the VLANs and sets them for WAN, but the VOIP VLAN, despite working DHCP, does not authenticate.

**Comment by billbarni** [comment]
I've been reading this post in OpnSense about IPTV (last reply):
https://forum.opnsense.org/index.php?topic=35933.msg175397#msg175397

Theoretically there needs to be a Bridge and also NAT to allow the VOIP Vlan to access both the port and the internet to be able to authenticate and communicate.

I've not been able to acomplish it. Not sure why. My Linksys PAP2T is never able to authenticate/communicate.
